README.md

```diff
@@ -26,13 +26,39 @@ OpenSearch Continous Integration is an open source CI system for OpenSearch and
 - Deploy stacks with `npm run cdk deploy`
 
 ## Deployment
+
```

Comment:
Please update the TOC as well, there is a [markdown linter](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in VSCode that I would highly advise to make this easy.